### DISCUSSION TITLE
Address to access

### DISCUSSION THREAD

**Discussion by Qmig** [trustwallet/wallet-core#3191] [Ideas & Feature request]
ETH
## Description

Name:

HD Derivation Scheme:
USDT 
Symbol:

Documentation:

Reason:

## Checklist

Finish check list [here](https://developer.trustwallet.com/wallet-core/newblockchain#blockchain-checklist).

**Comment by satoshiotomakan** [comment]
@Qmig could you please explain what's the purpose of the issue?
